src/Migrate/custom/New-AzMigrateDiskMapping.ps1

```diff
@@ -32,14 +32,18 @@ function New-AzMigrateDiskMapping {
         ${DiskID},
 
         [Parameter(Mandatory)]
+        [ValidateSet("true" , "false")]
+        [ArgumentCompleter( { "true" , "false" })]
         [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
         [System.String]
         # Specifies whether the disk contains the Operating System for the source server to be migrated.
         ${IsOSDisk},
 
         [Parameter(Mandatory)]
+        [ValidateSet("Standard_LRS", "Premium_LRS", "StandardSSD_LRS")]
+        [ArgumentCompleter( { "Standard_LRS", "Premium_LRS", "StandardSSD_LRS" })]
         [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
-        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Support.DiskAccountType]
+        [System.String]
```

Comment:
You don't need to change enum type to string. To add a validate set is enough. please refer to the following codes
https://github.com/Azure/azure-powershell/blob/8cd4f3a23f5b9db6dfbe7bcdb35964e4528872a8/src/MySql/custom/New-AzMySqlServer.ps1#L88